config/manager/kustomization.yaml

```diff
@@ -12,5 +12,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: quay.io/openstack-k8s-operators/glance-operator
-  newTag: latest
+  newName: quay.io/rhn_engineering_mbayer/glance-operator
+  newTag: 0.0.8
```

Comment:
this keeps getting merged in here every time i build an image.   ill take this out when we're ready

Reply:
+1, I think we should start consolidating the mariadb changes first, so we can rebase this patch (that in general looks good) with the new API calls provided by mariadb